codegen/templates/tchannel_endpoint.tmpl

```diff
@@ -266,7 +268,13 @@ func (h *{{$handlerName}}) redirectToDeputy(
 		},
 	)
 
-	success, respHeaders, err := client.Call(ctx, "{{.ThriftService}}", "{{$methodName}}", reqHeaders, req, res)
+	timeoutAndRetryOps := zanzibar.TimeoutAndRetryOptions{
+		OverallTimeoutInMs: 			(timeoutPerAttempt + zanzibar.DefaultBackOffTimeAcrossRetries) * time.Duration(maxAttempts),
```

Comment:
Can you extract this to `tchannel_client.go` and simply invoke from the template. We don't need to generate this much new code for all 600+ tchannel endpoint workflows.